resources/apigateway/helm-values.yaml

```diff
@@ -10,9 +10,18 @@ image:
   repository: custom-kong
   tag: 1.0.0
 
-ingressController:
```

Comment:
We can use a gateway solution that does not require feature gates to be activated